WMF Framework/CommonStrings.swift

```diff
@@ -238,12 +238,14 @@ public class CommonStrings: NSObject {
 //let menuLastEditedTitle: String
 //if let lastModified = lastModified {
 //    let days = NSCalendar.wmf_gregorian().wmf_days(from: lastModified, to: Date())
-//    menuLastEditedTitle = String.localizedStringWithFormat(WMFLocalizedString("page-last-edited",  language: language, value: "{{PLURAL:%1$d|0=Edited today|1=Edited yesterday|Edited %1$d days ago}}", comment: "Relative days since an article was last edited. 0 = today, singular = yesterday. %1$d will be replaced with the number of days ago."), days)
+//    menuLastEditedTitle = String.localizedStringWithFormat(WMFLocalizedString("page-last-edited",  language: language, value: "{{PLURAL:%1$d|0=Updated today|1=Updated yesterday|Updated %1$d days ago}}", comment: "Relative days since an article was last edited. 0 = today, singular = yesterday. %1$d will be replaced with the number of days ago."), days)
```

Comment:
I altered "Last updated" in the mock to "Updated" to simplify a bit, will confirm with Carolyn